### ISSUE TITLE
Bug with two 50 % slides when containScroll = true

### ISSUE THREAD

**Issue by niubsta** [davidcetinkaya/embla-carousel#30]
Hi!

First of all, let me say that I really like embla-carousel. Thank you for sharing your awesome work!!

There's one little bug I discovered when I have two slides with width: 50% and containScroll is set to true. It doesn't appear if I have only one slide with width: 100% or three slides with width: 33.33%.

Here's a demonstration of what I'm talking about: https://codesandbox.io/s/embla-carousel-loop-true-xgg95

It's actually easy to fix by setting the width to 49.99 % but as this is only a workaround, I wanted to report the issue.

Once again, thanks for your efforts!

**Comment by davidcetinkaya** [comment]
Hi Gerhard (@niubsta),
Thank you very much! I'm glad to hear that you like Embla Carousel 🎉.

Nice catch! Thanks for taking time to report this. I will look into this as soon as possible and I'll let you know how it goes when I have an update.

Best,
David

**Comment by davidcetinkaya** [comment]
Hello Gerhard (@niubsta),

I've released a fix for the bug you describe in version [2.5.3](https://github.com/davidcetinkaya/embla-carousel/releases/tag/v2.5.3). I'd very much appreciate if you could bump to the latest version like so:

```bash
yarn add embla-carousel@latest
```

or:

```bash
npm install embla-carousel@latest
```

...and confirm if [this fix](https://github.com/davidcetinkaya/embla-carousel/commit/402de846c0804e77200d9929dd809d4447ab1ce5) works.

Thank you in advance!
David

**Comment by niubsta** [comment]
Hello @davidcetinkaya ,

wow, that was quick - thank you so much!! I can confirm that the issue was solved by your fix!

Thanks again and kind regards,
Gerhard
